verticapy/machine_learning/memmodel/preprocessing.py

```diff
@@ -150,26 +296,103 @@ def __init__(self, min_: ArrayLike, max_: ArrayLike) -> None:
 
 class OneHotEncoder(InMemoryModel):
     """
-    InMemoryModel implementation of one-hot encoder.
+    :py:mod:`verticapy.machine_learning.memmodel.base.InMemoryModel`
+    implementation of one-hot encoder.
 
     Parameters
     ----------
+
     categories: ArrayLike
         ArrayLike  of the categories of  the different  features.
     column_naming: str, optional
         Appends  categorical  levels  to column  names  according
         to the specified method:
-        indices              : Uses integer  indices to represent
-                               categorical levels.
-        values/values_relaxed: Both methods use categorical level
-                               names.  If duplicate column  names
-                               occur,  the  function attempts  to
-                               disambiguate them by appending _n,
-                               where  n  is a zero-based  integer
-                               index (_0, _1,…).
+
+        - indices              : Uses integer  indices to represent
+            categorical levels.
+
+        - values/values_relaxed: Both methods use categorical level names.
+            If duplicate column  names occur,  the  function attempts  to
+            disambiguate them by appending _n, where  n  is a zero-based
+            integer index (_0, _1,…).
+
     drop_first: bool, optional
         If set to False, the first dummy of each category is
         dropped.
+
+    .. note::
+
+        :py:mod:`verticapy.machine_learning.memmodel` are defined
+        entirely by their attributes. For example, 'categories' to
+        encode defines a OneHotEncoder model. You can optionally
+        provide 'column naming' criteria and a 'drop_first' flag to
+        denote whether to drop first dummy of each category.
+
+    Examples
+    --------
+
+    **Initalization**
+
+    Import the required module.
+
+    .. ipython:: python
+
+        from verticapy.machine_learning.memmodel.preprocessing import OneHotEncoder
+
+    A OneHotEncoder model is defined by categories, column naming
+    criteria and drop_first flag. In this example, we will use the
+    following:
+
+    .. ipython:: python
+
+        categories = [["male", "female"], [1, 2, 3]]
+        drop_first = False
+        column_naming = None
+
+    Let's create a
+    :py:mod:`verticapy.machine_learning.memmodel.preprocessing.OneHotEncoder`
+    model.
+
+    .. ipython:: python
+
+        model_ohe = OneHotEncoder(categories, drop_first, column_naming)
```

Comment:
for consistency and ease of users to copy, can you combine it into one block i.e

``` 
model = OneHotEncoder(
        categories = categories,
        drop_first = False,
        column_naming = "values"
)

```